### ISSUE TITLE
bitbucket integration

### ISSUE THREAD

**Issue by richardreeve** [BES2016Workshop/CodePublication-and-citation#3]
Does Zenodo allow bitbucket integration, as that's where our university department keeps its code at the moment? If not, are there any plans to do that?

**Comment by AugustT** [comment]
Doesnt look like it does

**Comment by richardreeve** [comment]
No, but it may be in development, and if not it would be very useful for them to think about.
